src/coord/timestamp.rs

```diff
@@ -1481,22 +1490,26 @@ impl Timestamper {
                     let partitions = get_kafka_partitions(&kc.consumer, &kc.topic);
                     let partition_count = i32::try_from(partitions.len()).unwrap();
                     for p in partitions {
+                        let current_p_offset = cons
+                            .last_partition_offset
+                            .entry(PartitionId::Kafka(p))
+                            .or_insert(0);
```

Comment:
should this be `start_offset` instead of `0` ? (I haven't thought hard about it so could be wrong)

Reply:
Yes. Fixed